react-frontend/src/components/StyleComponents/htmlTags.js

```diff
@@ -23,6 +24,26 @@ export const BreadcrumbUL = styled.ul`
   margin: 0;
 `;
 
+export const DigitalFrameworkP = styled.p.attrs({
+  className: 'digitalParagraph',
+})`
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Please change **font-family** to `‘BC Sans’, ‘Noto Sans’, Verdana, Arial, sans-serif;`